packages/search/addon/services/navi-search/navi-report-search-provider.js

```diff
@@ -0,0 +1,47 @@
+import NaviBaseSearchProviderService from './navi-base-search-provider';
+
+export default class NaviReportSearchProviderService extends NaviBaseSearchProviderService {
+  /**
+   * @property name
+   * @override
+   */
+  name = 'reportSearchProvider';
+
+  /**
+   * @property niceName
+   * @override
+   */
+  niceName = 'Report Search Provder';
+
+  /**
+   * @property associatedComponent
+   */
+  associatedComponent = 'navi-report-search-result';
+
+  /**
+   * @method search
+   * @override
+   * @param {Object} searchParams
+   * @param {Object} author
+   * @returns {Promise} promise
+   */
+  search(searchParams, author) {
```

Comment:
Can this function be run with just an author property? What if searchParams is an empty object?

Reply:
it now can, it will return the reports that are authored by the specified user, added a test for it as well.